src/crypto/types.ts

```diff
@@ -13,3 +13,23 @@ export interface ICrypto {
 
   toOutputScript(address: string): Buffer;
 }
+
+export type Currency =
```

Comment:
I think you meant to use this type for the currency property but instead you used string ?

Reply:
Fixed. The code can be reviewed again. Thanks